pom.xml

```diff
@@ -82,14 +77,34 @@
             <version>1.3.0_1.1.1_0.0.6</version>
         </dependency>
         <dependency>
+            <groupId>it.cnr.isti.hpc</groupId>
+            <artifactId>json-wikipedia</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+        <dependency>
             <groupId>org.dbpedia.spotlight</groupId>
-            <artifactId>spotlight</artifactId>
+            <artifactId>core</artifactId>
             <version>0.7</version>
         </dependency>
         <dependency>
+            <artifactId>hpc-utils</artifactId>
             <groupId>it.cnr.isti.hpc</groupId>
-            <artifactId>json-wikipedia</artifactId>
-            <version>1.0.0</version>
+            <version>0.0.5</version>
+        </dependency>
+        <dependency>
+            <groupId>info.bliki.wiki</groupId>
+            <artifactId>bliki-core</artifactId>
+            <version>3.0.16</version>
```

Comment:
This bliki dependency is coming from JsonWikipedia.
They have added the jar inside the lib folder.
Check this 
https://github.com/naveenmadhire/json-wikipedia-dbspotlight/tree/master/lib/info/bliki/wiki